Mathlib/Probability/Independence/Basic.lean

```diff
@@ -731,6 +731,31 @@ lemma iIndepFun_precomp_of_bijective {g : ι' → ι} (hg : g.Bijective) :
   mp := .of_precomp hg.surjective
   mpr := .precomp hg.injective
 
+theorem iIndepFun_iff_pi_map_eq_map [Fintype ι] [IsProbabilityMeasure μ]
```

Comment:
Can you move that next to `ProbabilityTheory.indepFun_iff_map_prod_eq_prod_map_map`?